### PR TITLE
Small UI Fixes

### PR DESCRIPTION
**Pull request by francisschmaltz**
This PR contains small UI fixes for Sourcegraph.

### CODE REVIEW

**Comment by francisschmaltz** [comment]
- Corrects typo in explore link
- Aligns calculation of search with search results
- Changes "Self-hosted" to "Install Sourcegraph"
- Makes "About Sourcegraph" user nav link into a group.

File: packages/webapp/src/search/results/SearchResultsInfoBar.scss [hunk 1]
```diff
@@ -4,6 +4,11 @@
     justify-content: space-between;
     color: #93a9c8;
     padding-bottom: 0.25rem;
+	padding-left: .625rem;
+
+	svg {
```
Comment (felixfbecker) [inline packages/webapp/src/search/results/SearchResultsInfoBar.scss]: Why does this need to be a child selector (as opposed to a class)?

Comment (francisschmaltz) [reply packages/webapp/src/search/results/SearchResultsInfoBar.scss]: It doesn't **have** to be but I used it incase the class might change.

Comment (felixfbecker) [reply packages/webapp/src/search/results/SearchResultsInfoBar.scss]: See https://sourcegraph.sgdev.org/github.com/sourcegraph/sourcegraph/-/blob/doc/dev/web_app.md#L37

**Comment by francisschmaltz** [comment]
- Fixes close button on panel

**Pull request merged**